packages/genesys-spark-components/src/components/stable/gux-tooltip-title/gux-tooltip-title.tsx

```diff
@@ -73,11 +73,12 @@ export class GuxTooltipTitle {
   onMutation(): void {
     this.titleName = this.setTooltipTitleText();
     this.checkForTooltipHideOrShow();
+  }
 
-    // Handle checking if the tooltip should hide/show for dynamic content
-    afterNextRenderTimeout(() => {
-      this.checkForTooltipHideOrShow();
-    }, 500);
+  @OnResize()
+  onResize(): void {
+    this.titleName = this.setTooltipTitleText();
+    this.checkForTooltipHideOrShow();
   }
```

Comment:
Apparently a slot change doesn't always hit the OnMutation, but this case does hit the OnResize